lms/djangoapps/certificates/api.py

```diff
@@ -728,16 +723,44 @@ def create_certificate_invalidation_entry(certificate, user_requesting_invalidat
     return certificate_invalidation
 
 
-def get_certificate_invalidation_entry(certificate):
+def get_certificate_invalidation_entry(generated_certificate, invalidated_by=None, notes=None, active=None):
     """
-    Retrieves and returns an certificate invalidation entry for a given certificate id.
+    Retrieve a certificate invalidation entry for a specified certificate.
+
+    Optionally filter the invalidation entry by who invalidated it, notes, and whether the invalidation is active.
```

Comment:
```suggestion
    Optionally filter the invalidation entry by one or more of who invalidated it, notes, or whether the invalidation is active.
```
This is less a suggestion than it is asking for clarification. This is what it looks like is going on in the code, am I correct?